google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -300,6 +302,10 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      UsageTrackerProvider.getInstance()
+          .trackEvent(GctTracking.APP_ENGINE_SUPPORT_ADDED)
```

Comment:
What I meant about being explicit about tracking new projects wizard usages is to add the ping directly on the method that the wizard uses (not on the side effect of adding ae support). It could also use a top event instead of a label.

Didn't you mention that this method gets invoked for imports? If so, then the current logic would incorrectly count imports as new projects, no?